### PR TITLE
chore: modify module-checks to support Typescript modules

### PR DESCRIPTION
**Pull request by LeonLuttenberger**
## Describe your changes

- Changed `module-checks.yml` to support Typescript in addition to Python
- Updated `validate.sh` to support running `eslint`
- Added an eslint configuration to the `fmops` module
- Reformatted the `fmops` module to adhere to the standards

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by kukushking — APPROVED** [review]
Looks good to me. Is this ready to go?

**Comment by LeonLuttenberger** [comment]
I haven't tested it yet with any Typescript module. I was gonna create an example module, but it might be more efficient to just merge #13 and then use that to test this PR.

**Comment by kukushking** [comment]
Deal.

File: modules/fmops/sagemaker-jumpstart-fm-endpoint/.eslintrc [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (kukushking) [inline modules/fmops/sagemaker-jumpstart-fm-endpoint/.eslintrc:1]: Thanks!

**Review by kukushking — APPROVED** [review]
Thanks! Awesome, we now support typescript! 😍

**Pull request merged**